.changeset/perfect-cobras-bake.md

```diff
@@ -0,0 +1,9 @@
+---
+'@backstage/plugin-jenkins-backend': minor
+'@backstage/plugin-jenkins': minor
+'example-app': minor
+---
+
+Added JobRunTable in EntityPage
+Added JobRunTable component in Jenkins frontend plugin.
+Added new Route and extended Api to get buildJobs.
```

Comment:
Can we please update the Jenkins plugin's `README.md` with the following:

- images showing the new table that has been added with a description of its functionality
- instructions on how to add this table so that people using the plugin know how to use it

Reply:
@awanlin  thank you for your feedback, i have made the changes as per the feedback